websites/H/Home Assistant/presence.ts

```diff
@@ -0,0 +1,131 @@
+const presence = new Presence({
+    clientId: "812627990382772224"
+}), browsingStamp = Math.floor(Date.now() / 1000);
+  
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "home_assistant_logo" //banner
+  }; 
+  presenceData.startTimestamp = browsingStamp;
+  
+
+  function homeassistant(state: string) {
+    presenceData.details = "Viewing";
+    presenceData.state = state;
+  }
```

Comment:
It isn't good option to do it...

Reply:
How can we change this? Presencedata details is always "Viewing"

Reply:
It doesn't always mean that it must be "Viewing" everytime??